pkg/controller/migmigration/annotation.go

```diff
@@ -36,6 +37,11 @@ const (
 	ResticPvVerifyAnnotation = "backup.velero.io/verify-volumes" // comma-separated list of volume names
 )
 
+// Disables the internal image copy
+const (
+	DisableImageCopy = "openshift.io/disable-image-copy"
```

Comment:
For future reference, we should be namespacing our annotations with `migration.openshift.io`